lib/Fmt_ast.ml

```diff
@@ -3288,7 +3289,7 @@ and fmt_constructor_declaration c ctx ~max_len_name ~first ~last:_ cstr_decl
       | Pcstr_tuple x -> List.is_empty x
       | Pcstr_record x -> List.is_empty x
     in
-    let len_around = if is_symbol_id txt then 4 else 0 in
+    let len_around = if String_id.is_symbol txt then 4 else 0 in
```

Comment:
It's already useful to organize `Ast` into better-scoped modules, thanks for this. I agree with @Julow is that it's a good occasion to add some tests if you can.

If you want to try to take it a step further (not in this PR!), I noticed that this API is returning a lot of booleans. In the spirit of tell-dont-ask, you can try to build an abstraction that encapsulates the boolean logic. In this example, something like: `String_id.of_name txt |> String_id.padding ~max_len_name`. If this goes well, `is_symbol` might not even need to be exposed. This kind of things. There's a balance to be found, it can be tricky.

Reply:
That's a good idea, I will see better what we can do with it when the file has been cleaned up (more like purified)